### ISSUE TITLE
Crontab jobs for sending trending Topics to Users

### ISSUE THREAD

**Issue by namangupta01** [publiclab/plots2#2091]
### Please describe the problem (or idea)

We can use crontab jobs to send mails to users about trending topics and for the other stuff just like we got mail from twitter about treading topics or about people. I think it would would be great if users get to know about the trending topics on mail. What do you say @jywarren @icarito @ryzokuken ?

Timing of sending of the mail can be pre set so that we know when we are going to get the mail.

**Comment by jywarren** [comment]
Hi, I like this idea a lot -- we're actually working on a related set of issues in #2104, and to do this kind of thing we first need a kind of "digest" of followed content, which is the subject of #1119. 

After that, we're going to try to send this digest each day by email in https://github.com/publiclab/plots2/issues/1584

We could definitely use your help with some of these! I wonder if you could start working based on #2027 but developing an email template that can be triggered in the SubscriptionMailer that would send the digest!

We could rename this issue `Develop email template and mailer for followed content digests` and -- even without ActiveJob (cron) triggers, we can start testing this feature by putting a button on /subscriptions that says `email me the past week's followed content` 

This way we can test this out on production a bit without worrying about sending a million emails by mistake :-P

We'd LOVE your help with this! 👍

**Comment by jywarren** [comment]
I also LOVE that you thought of this in parallel! To me, that means it's a really good idea!

**Comment by jywarren** [comment]
I think this can be merged into #2104 now! Please chime in if there's anything missing over there though!
